include/dbm/fed.h

```diff
@@ -190,11 +190,11 @@ namespace dbm
         /// @return the dimension of this DBM.
         cindex_t getDimension() const;
 
-        /// @return string representation of the
-        /// constraints of this DBM. A clock
-        /// is always positive, so "true" simply means
-        /// all clocks positive.
-        std::string toString(const ClockAccessor&, bool full = false) const;
+        /// Prints a textual representation of the DBM constraints.
+        /// A Clock is always positive, so "true" simply means alll clocks positive.
```

Comment:
This is no longer true as clocks can have negative values

Reply:
This is still true in DBM context.
![image](https://user-images.githubusercontent.com/4868894/205597540-ca4b6f1f-f088-4e41-bf81-c9229393f440.png)